gui/src/app/Stanc/stancWorker.ts

```diff
@@ -19,30 +24,37 @@ try {
   console.error(e);
 }
 
+// helper alias for type safety
+const postReply = (message: StancReplyMessage) => self.postMessage(message);
```

Comment:
also has the nice property of clarifying that you're responding to something. Elegant!